internal/versioning/version.go

```diff
@@ -0,0 +1,9 @@
+package versioning
+
+var (
+	ApplicationVersion string
+
+	// XXX: Bump this value only when there are protocol changes that makes the oracle
+	// incompatible between version!
+	ProtocolVersion = `v0.5.1.2`
```

Comment:
as a note @j2d3 - that's what we have to stick now with the current bootnodes which are deployed, right? just double checking here